### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by mehcode** [chuckhendo/project-quick-open#18]
**Atom Version**: 0.199.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: linux 3.19.0-16-generic
**Thrown From**: [project-quick-open](https://github.com/chuckhendo/project-quick-open) package, v0.4.4

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /home/ryan/.atom/packages/project-quick-open/lib/project-quick-open-view.coffee:29

TypeError: undefined is not a function
  at ProjectQuickOpenView.module.exports.ProjectQuickOpenView.confirmed (/home/ryan/.atom/packages/project-quick-open/lib/project-quick-open-view.coffee:29:26)
  at ProjectQuickOpenView.module.exports.SelectListView.confirmSelection (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:238:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:519:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:354:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:178:20)

```

### Commands

```
     -3:38.6.0 settings-view:open (atom-text-editor.editor.is-focused)
  2x -3:32.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:58 editor:move-to-beginning-of-word (atom-text-editor.editor.is-focused)
  3x -2:57.8.0 editor:select-to-end-of-word (atom-text-editor.editor.is-focused)
     -2:56.8.0 project-find:show (atom-text-editor.editor.is-focused)
     -2:56 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:41.7.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -2:41.7.0 core:cancel (atom-text-editor.editor.mini.is-focused)
  2x -2:41 core:close (ol.results-view.list-tree.focusable-panel.has-collapsable-children)
     -0:06.6.0 core:backspace (atom-text-editor#project-quick-open.projectPaths.editor.mini.is-focused)
     -0:03.1.0 core:close (div.settings-view.pane-item)
     -0:02.4.0 project-quick-open:toggle (atom-text-editor.editor.is-focused)
     -0:01.7.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:01.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.4.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -0:00.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {},
  "project-quick-open": {
    "projectPaths": "~/Workspace",
    "openProjectsInSameWindow": true
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.0.0
editorconfig, v1.0.0
file-icons, v1.5.5
language-generic-config, v0.2.0
minimap, v4.8.0
project-quick-open, v0.4.4
project-view, v0.1.2

# Dev
No dev packages
```

**Comment by mehcode** [comment]
To be clear; this works normally. Just thought I'd submit the issue as it breaks when atom is run with `atom --one`

**Comment by chuckhendo** [comment]
@mehcode I believe this is fixed with v0.5.0. Would you mind confirming that?

**Comment by mehcode** [comment]
Looks like it. Thank you.
